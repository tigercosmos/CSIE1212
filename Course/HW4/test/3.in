bbcacbbbcb
100
3 a
2 b
3 bb
2 c
3 cc
3 c
3 bc
2 c
2 c
1 b
3 bccc
1 c
1 b
3 ca
1 b
1 b
1 b
3 b
2 c
3 cacbb
3 b
3 b
1 a
1 a
1 a
3 c
2 a
1 c
3 a
1 c
3 b
2 b
1 c
1 b
3 bbbca
3 bb
1 a
2 a
2 a
2 b
1 a
3 a
2 a
1 a
1 c
1 b
3 b
2 a
3 bcbbb
3 baa
2 a
3 b
2 a
2 a
3 b
3 a
1 a
1 c
1 a
3 b
1 c
3 baaba
3 bb
2 a
3 aabcc
3 aa
2 c
1 a
2 a
2 b
2 b
2 b
1 a
1 b
2 b
1 c
3 b
2 a
2 b
2 a
2 c
3 a
1 b
1 c
2 a
3 c
3 a
3 aabb
3 b
1 b
1 a
2 b
3 b
1 c
3 a
1 a
1 c
3 aa
2 c
2 c
