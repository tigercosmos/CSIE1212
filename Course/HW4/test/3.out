1
4
0
4
3
1
1
12
1
12
12
8
5
12
1
8
9
16
1
2
16
16
15
16
1
8
1
10
21
23
18
24
1
23
25
25
11
